src/features/home-page/homepage-nav-bottom/homepage-nav-bottom.js

```diff
@@ -20,7 +22,9 @@ class HomepageNavBottom extends Component {
             <span className="menu__weather">Weather</span>
           </li>
           <li className="menu__item menu__item-nobg">
-            <span className="menu__time">Time</span>
+            <span className="menu__time">
+              <ShowTime />
```

Comment:
@IraLukianovets  this way component will render only time, (need also information about date)